Src/POI.DiscordDotNet/Services/Implementations/DiscordStarboardService.cs

```diff
@@ -40,30 +40,36 @@ public void Cleanup(IDiscordClientProvider discordClientProvider)
 
 	private async Task OnReactionAdded(DiscordClient sender, MessageReactionAddEventArgs args)
 	{
-		//TODO: Check if message was not cached
 		//TODO: Check on emoji ID, not name
 		//TODO: Figure out a way to get the display name of the user outside the guild...
```

Comment:
This last todo isn't really applicable as the starboard can't be used outside guilds